service/RumReport.js

```diff
@@ -151,9 +151,8 @@ function Compat() {
 		})
 		.then(results => {
 			return dbconn.end().then(() => {
-				return Promise.all(results[0].map(rec => {
-					return polyfillio.describePolyfill(rec.feature_name)
-					.then(polyfill => {
+				return results[0].map(rec => {
+					const polyfill = polyfillio.describePolyfill(rec.feature_name);
```

Comment:
Should this be using the sources lib directly rather than the external API, I wonder?